Controller/ElasticsearchController.php

```diff
@@ -559,12 +559,8 @@ public function searchApiAction(Request $request)
             //http://blog.alterphp.com/2012/08/how-to-deal-with-asynchronous-request.html
             $request->getSession()->save();
 
-            /** @var Client $client */
-            $client = $this->getElasticsearch();
-
             $this->getLogger()->debug('Before search api');
-            $results = $client->search($params);
-
+            $results = $this->elasticsearchClient->searchByRequest($searchRequest)->toArray();
```

Comment:
should you not use the $body variable in the search?

Reply:
The body is in the request object that we are passing?

Reply:
I dont see the line where this happens?

Reply:
checked it again, where you create the body variable, until this line. no set is happening on the SearchRequest

Reply:
Omg you're right sorry! Will fix it

Reply:
Fixed